scripts/config.json

```diff
@@ -7,6 +7,7 @@
         "search_service_name": "<search service name to use or create>",
         "index_name": "<index name to use or create>",
         "chunk_size": 1024,
+        "token_overlap": 128,

```

Comment:
Update the example in the README with the same?

Reply:
Good idea. Will do.

Reply:
Done